src/app/clusters/level-control/level-control.cpp

```diff
@@ -52,6 +52,8 @@
 
 #include <app/reporting/reporting.h>
 #include <platform/CHIPDeviceConfig.h>
+#include <platform/DeviceControlServer.h>
```

Comment:
Where do we use DeviceControlServer?

Reply:
Previous code in this PR used it. Removed and included `platform/CHIPDeviceLayer.h`